README.md

```diff
@@ -1,17 +1,81 @@
 # Repository *jsramverk-train-controller*
+
 This is the repository for the course *jsramverk* (h23).
 It is maintained by *elmo22* and *poak22*.
 
+The application can be found here: [Train-controller](https://www.student.bth.se/~elmo22/train/). And the backend can be found here: [Backend](https://jsramverk-train-poak22-elmo22.azurewebsites.net/).
+
+## How to use this repo
+
 ```
-# To initialze the application you need to run the following script from root folder in repo.
+# To initialze the application, run the following script from root folder in repo.
 bash setup_app.bash
 ```
 
-You also need a .env file in the /backend folder with API-key, see *.env.example* for structure.
+A a .env file is needed in the /backend folder with API-key, see *.env.example* for structure.
+
+When developing this repo both the backend and frontend locally needs to be started locally. This will run backend and frontend in the development environment, which won't effect the database used in the deployed application. When starting frontend in development the URL 'localhost:1337' is used to fetch data, where backend then uses it's own database 'development' in the Atlas Cloud.
+
+```
+# Run backend from /backend folder
+npm run start-dev
+```
+
+```
+# Run frontend from /frontend-vue folder
+npm run dev
+```
+### Reset development database
+
+If needed during development, the development database can be reset.
+
+```
+# To reset the development database, stand in /backend folder.
+npm run dev-reset-db
+```
+
+### Tests
+
+Test can be run for both backend and frontend.
+
+#### Backend
+
+Mocha together with chai is used for backend testing. Testing is done on a it's own database called 'test' that is set-up in the Atlas Cloud. The command below is also the command used in github actions for continuous integration.
+
+```
+# Stand in /backend folder.
+npm run test
+```
+
+#### Frontend (Vue)
+
+For frontend testing cypress is used for end to end testing. Testing uses the API and also the 'test' database in the Atlas Cloud. Therefor the backend needs to be started locally in 'test-mode'. To run the test the commands below is needed to be run, which is also the way it's done during continuous integration.
```

Comment:
förbättringsförslag: ... The backend needs to be started locally in 'test-mode'.